packages/create-block/lib/templates/plugin/$slug.php.mustache

```diff
@@ -42,6 +42,14 @@ if ( ! defined( 'ABSPATH' ) ) {
  * @see https://developer.wordpress.org/reference/functions/register_block_type/
  */
 function {{namespaceSnakeCase}}_{{slugSnakeCase}}_block_init() {
+
+	// Register the block metadata.
+	wp_register_block_metadata_collection(
```

Comment:
This method will exist only in WordPress 6.7 so we would have to call it conditionally as long as the minimum required version for WP is set to 6.6.

By the way, @felixarntz opened a proposal https://core.trac.wordpress.org/ticket/62267 to add an option to this function to automatically register blocks listed in the manifest file. With that, we would no longer need to add `register_block_type( __DIR__ . '/build' );` for every block type included in the plugin. Eventually (WordPress 6.8?), the following should become the only code required no matter how many block types the plugin contains:

```php
// Registers the block types metadata.
wp_register_block_metadata_collection(
	__DIR__ . '/build'
	__DIR__ . '/build/blocks-manifest.php',
	array( 'auto_register' => true )
);

Reply:
>This method will exist only in WordPress 6.7 so we would have to call it conditionally as long as the minimum required version for WP is set to 6.6.

I'm sorry, I had added a check but didn't push up the changes. I went with a simple `function_exists` call.

>With that, we would no longer need to add register_block_type( __DIR__ . '/build' ); for every block type included in the plugin.

That would be amazing. It's pretty easy to forget to go back and add that line for a new block.